libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -38,6 +44,14 @@ export class BookSearchComponent implements OnInit {
     this.store.select(getAllBooks).subscribe(books => {
       this.books = books;
     });
+
+    this.subscription = this.actions$
```

Comment:
This is _extremely_ unusual. I wouldn't normally recommend watching the action stream in a component. Do this in an effect class instead.

Reply:
I wasn't sure if it was against best practice to watch an action stream in a component. I moved this logic to the effect class.